src/test/java/com/example/lures/LureMapperTest.java

```diff
@@ -0,0 +1,120 @@
+package com.example.lures;
+
+import com.github.database.rider.core.api.dataset.DataSet;
+import com.github.database.rider.core.api.dataset.ExpectedDataSet;
+import com.github.database.rider.spring.api.DBRider;
+import org.junit.jupiter.api.Test;
+import org.mybatis.spring.boot.test.autoconfigure.MybatisTest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DBRider
+@MybatisTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class LureMapperTest {
+
+    @Autowired
+    LureMapper lureMapper;
+
+    @Test
+    @DataSet(value = "datasets/lures.yml")
+    @Transactional
+    public void すべてのルアーデータが取得できること() {
+        List<Lure> lureList = lureMapper.findAll();
+
+        assertThat(lureList)
+                .hasSize(4)
+                .contains(
+                        new Lure(1, "Balaam300", "Madness", 300, 168),
+                        new Lure(2, "JointedCRAW178", "GANCRAFT", 178, 56),
+                        new Lure(3, "SILENTKILLER175", "deps", 175, 70),
+                        new Lure(4, "BULLSHOOTER160", "deps", 160, 98)
+                );
+    }
+
+    @Test
+    @DataSet(value = "datasets/lures.yml")
+    @Transactional
+    public void 引数に指定した文字列を含むルアーデータが返されること() {
+        List<Lure> lureList = lureMapper.findLures("join");
+
+        assertThat(lureList)
+                .hasSize(1)
+                .contains(
+                        new Lure(2, "JointedCRAW178", "GANCRAFT", 178, 56)
+                );
+    }
+
+    @Test
+    @DataSet(value = "datasets/lures.yml")
+    @Transactional
+    public void 存在するIDを指定した場合に該当するルアーデータが返されること() {
+        Optional<Lure> lureList = lureMapper.findById(1);

```

Comment:
list ではないので変数名変えたいですね。
```suggestion
        Optional<Lure> lure = lureMapper.findById(1);

```

Reply:
d284046fed5358e65d29d6329461c8038da69207
`lureList`から`lure`に修正しました。